test/go-tests/test_webhook.go

```diff
@@ -188,12 +188,12 @@ spec:
             name: "my-webhook-k8s-secret"
             key: "my-key"
       requests:
-        - url: http://shipyard-controller:8080/v1/project/{{.data.project}}
+        - url: http://google.com
```

Comment:
Let's do some self advertise 😄 
```suggestion
        - url: http://keptn.sh
```